### DISCUSSION TITLE
Is it just me or the Github Actions useful area is shrinked?

### DISCUSSION THREAD

**Discussion by onhate** [community/community#26835] [Actions]
<p>I have a macbook 16 inches which has a pretty good resolution and even though I don’t feel like I can get the best of Github Actions logs, more than half of the page is useless for the context and if you try to expand some log you can just see a few lines.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/df96f37b2cf8835c417f67f303625c6490c6636b.png" data-download-href="/uploads/short-url/vTXT0bSlUpoc4d5BkE5T9ktcd6j.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/980905/181100239-c86465f1-9c0e-4f70-ae50-0bbf07314ac1.png" alt="image" data-base62-sha1="vTXT0bSlUpoc4d5BkE5T9ktcd6j" width="690" height="381" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/df96f37b2cf8835c417f67f303625c6490c6636b_2_690x381.png, https://aws1.discourse-cdn.com/github/optimized/2X/d/df96f37b2cf8835c417f67f303625c6490c6636b_2_1035x571.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/d/df96f37b2cf8835c417f67f303625c6490c6636b_2_1380x762.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/df96f37b2cf8835c417f67f303625c6490c6636b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">2878×1592 418 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/onhate">@onhate</a>,</p>
<p>Now, the log for GitHub Actions supports <strong>full-screen view</strong> mode.<br>
At the upper right corner of the log screen, click “<strong>. . .</strong>” &gt; “<strong>Show full screen</strong>”, the log screen will switch to full screen mode.<br>
<img src="https://user-images.githubusercontent.com/46307996/181100240-20d2f5b0-c228-4f9f-a3fb-c33adda1927c.png" alt="options" data-base62-sha1="zWqOsbNNPRT0hERrZeMXfok7rYM" width="331" height="271"></p>
<p>More details about the new logs experience, you can see “<a href="https://github.blog/2020-09-23-a-better-logs-experience-with-github-actions/" rel="noopener nofollow ugc"><strong>A better logs experience with GitHub Actions</strong></a>”.</p>
